libraries/lib-wave-track/Sequence.cpp

```diff
@@ -144,7 +135,9 @@ bool Sequence::ConvertToSampleFormat(sampleFormat format,
       // no change
       return false;
 
-   if (mBlock.size() == 0)
+   const size_t blockCount = mBlockCount.load(std::memory_order_relaxed);
```

Comment:
In that case `.load` might not be necessary, but won't harm either

Reply:
I agree, but let's keep it, for who knows where this code will end up eventually.